app/src/lib/components/Intro/IntroForm.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+import { IntroOptionType } from './Intro';
+
+type IntroDesProps = {
```

Comment:
```suggestion
interface IntroDesProps = {
```
typescript 자체에서 interface를 권장한다고 합니다.
선언 병합 때문이라고 하는데 type은 선언 병합이 불가능하지만 interface는 선언 병합이 가능합니다.

선언 병합이 가능하다는 것은 동일한 이름으로 interface를 여러번 선언해도 컴파일 시점에 합칠 수 있다는 것을 의미합니다. 때문에 우리처럼 오픈API를 배포할 때에는 interface가 적절해 보입니다. 
또 그와 별개로 typescript에서 특별한 경우가 아니면 interface 사용을 권장한다고 하네요. 

[type vs interface](https://medium.com/humanscape-tech/type-vs-interface-%EC%96%B8%EC%A0%9C-%EC%96%B4%EB%96%BB%EA%B2%8C-f36499b0de50)